core/src/Kokkos_GraphNode.hpp

```diff
@@ -246,6 +247,51 @@ class GraphNodeRef {
                       std::forward<Functor>(functor));
   }
 
+#if defined(KOKKOS_ENABLE_CUDA) ||                                           \
+    (defined(KOKKOS_ENABLE_HIP) && defined(KOKKOS_IMPL_HIP_NATIVE_GRAPH)) || \
+    (defined(KOKKOS_ENABLE_SYCL) && defined(SYCL_EXT_ONEAPI_GRAPH))
+  /// The parameter @p exec can be used for multi-GPU graphs.
+  /// The captured nodes will be assigned to the device associated with @p exec.
+  /// @todo Support label.
```

Comment:
Should we add a few sentences here to say what this function does? E.g. 

> Capture the backend operations that calling the passed function object with the passed execution space instance enqueues into the underlying stream.

Also, should we add "warnings" to say that it's the user’s responsibility to ensure that no fences and no other backend operations are enqueued during the capture? And say a word about how the functor is stored in the node to ensure the lifetime of the needed data?

Reply:
I've updated the wording a bit.

Reply:
Done, OK for you ?